Guide refactoring and new tutorials (#2579)
informalsystems/ibc-rs: guide/src/tutorials/local-chains/add-a-new-relay-path.md

> __WARNING__: In production, do not create clients, connections or channels between two chains before checking that a client/connection/channel does not already fulfill the same function.

---

## Identifiers

A chain allocates identifiers when it creates clients, connections and channels. These identifiers can subsequently be used to refer to existing clients, connections and channels.

> NOTE: If you want to ensure you get the same identifiers while following the tutorials, run each of the commands in this page only __once__ or reset the chains as instructed in section [Start local chains](./start-local-chains.md#reset-your-configuration-and-start-the-chains). 

Chains allocate identifiers using a chain specific allocation scheme. Currently, *cosmos-sdk* implementation uses the following identifiers:

- `07-tendermint-<n>` for tendermint clients.
- `connection-<n>` for connections.
- `channel-<n>` for channels.

> It is possible for two chains to use the same identifier to designate two different objects. e.g. `channel-0` does not designate the same channel on the Hub and on Osmosis or Juno.

Author: Indeed. I just want to emphasize that every identifier is local to a chain.

> It is possible for two chains to use the same identifier to designate two different objects. e.g. `channel-0` does not designate the same channel on the Hub and on Osmosis or Juno.

Reviewer: Let's change it to something like this: 
```suggestion
It is possible for two chains to use the same identifier to designate two different objects. For example, two different channels, one on the Hub and one on Osmosis, can both be designated with the `channel-0` identifier. 
```
